File: 프로그래머스/unrated/181856. 배열 비교하기/배열 비교하기.py
def solution(arr1, arr2):
    # 조건 1
    if len(arr1) > len(arr2):
        return 1
    elif len(arr1) < len(arr2):
        return -1
    
    # 조건 2
    if sum(arr1) > sum(arr2):
        return 1
    elif sum(arr1) < sum(arr2):
        return -1
    else:
        return 0

File: 프로그래머스/0/181856. 배열 비교하기/배열 비교하기.java
import java.util.*;

class Solution {
    public int solution(int[] arr1, int[] arr2) {
        int answer = Integer.compare(arr1.length, arr2.length);

        if (answer == 0) {
            return Integer.compare(Arrays.stream(arr1).sum(), Arrays.stream(arr2).sum());
        }

        return answer;
    }
}